src/utils/lang/index.js

```diff
@@ -92,6 +92,13 @@ export function isString(val) {
   return typeof val === 'string' || val instanceof String;
 }
 
+/**
+ * Checks if a given value is a string.
+ */
+export function isNumber(val) {
+  return typeof val === 'number' || val instanceof Number;
+}
+
```

Comment:
Why not using isFinite? It's the same validation we use somewhere else as probably having a number that results in NaN or Infinity is not expected by our customers but rather an issue on a calculation.